profiler/options.go

```diff
@@ -207,6 +210,7 @@ func defaultConfig() (*config, error) {
 		tags:              []string{fmt.Sprintf("process_id:%d", os.Getpid())},
 		deltaProfiles:     internal.BoolEnv("DD_PROFILING_DELTA", true),
 		logStartup:        true,
+		cmemprofRate:      2 * 1024 * 1024,
```

Comment:
NIT: Maybe use `cmemprof.DefaultSamplingRate` as a value here to avoid scattering the default value in various places that might get out of sync and confusing?